presto-native-execution/presto_cpp/main/PrestoTask.cpp

```diff
@@ -473,6 +473,8 @@ protocol::TaskInfo PrestoTask::updateInfoLocked() {
     taskRuntimeStats.insert(
         {"averageOutputBufferWallNanos",
          fromMillis(outputBufferStats.averageBufferTimeMs)});
+    taskRuntimeStats["numTopOutputBuffers"].addValue(
```

Comment:
nit: it would be nice to include this change in PR description and commit message.